vectordraw/templates/html/vectordraw.html

```diff
@@ -0,0 +1,130 @@
+{% load i18n %}
+<div class="vectordraw_block">
+
+  <h2>{{ self.display_name }}</h2>
+
+  {% if self.description %}
+    <div class="vectordraw-description">
+      {{ self.description|safe }}
```

Comment:
@nedbat We allow authors to format the description using a rich editor (TinyMCE), so it may contain HTML tags. (This is done by [initalizing the field with `multiline_editor="html"`](https://github.com/open-craft/xblock-vectordraw/blob/initial-implementation/vectordraw/vectordraw.py#L45) and [attaching a TinyMCE instance to the field](https://github.com/open-craft/xblock-vectordraw/blob/initial-implementation/vectordraw/public/js/studio_edit.js#L44) on the client.)